Actions/Copy.cs

```diff
@@ -1,9 +1,9 @@
 ﻿using UnityEditor;
-using UnityEngine.Experimental.EditorVR.Utilities;
 
 namespace UnityEngine.Experimental.EditorVR.Actions
 {
 	[ActionMenuItem("Copy", ActionMenuItemAttribute.kDefaultActionSectionName, 5)]
+	[Tooltip("Copy")]
```

Comment:
I was also considering having Action implement IToolTip and just hardcoding the text as a property. We probably want to still allow for the item name and tooltip text to be different, so I'm also amenable to an optional string parameter.